opensrp-anc/src/main/java/org/smartregister/anc/library/model/ContactVisit.java

```diff
@@ -149,7 +149,7 @@ private void updateEventAndRequiredStepsField(String baseEntityId, PartialContac
                     }
 
                     //process attention flags
-                    ANCFormUtils.processRequiredStepsField(facts, formObject);
+                    org.smartregister.anc.library.util.ANCFormUtils.processRequiredStepsField(facts, formObject);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.util.ANCFormUtils.processRequiredStepsField' due to existing import 'org.smartregister.anc.library.util.ANCFormUtils'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5257516)